syntax = "proto3";

message EchoRequest {
  string name = 1;
}

message EchoReply {
  string message = 1;
}

service EchoService {
  rpc SayHello(EchoRequest) returns (EchoReply) {}
}
